packages/app-page-builder/src/editor/plugins/elementSettings/action/ActionSettings.tsx

```diff
@@ -176,7 +193,32 @@ const ActionSettingsComponent: React.FC<ActionSettingsPropsType> = ({
                                         </Bind>
                                     </Wrapper>
                                 </>
-                            ) : (
+                            )}
+                            {actionType === "scrollToElement" && (
+                                <>
+                                    <Bind name={"scrollToElement"}>
+                                        {({ value, onChange }) => (
+                                            <Wrapper
+                                                label={"Element ID"}
+                                                containerClassName={classes.gridClass}
+                                            >
+                                                <SelectField
+                                                    value={value}
+                                                    onChange={onChange}
+                                                    placeholder={"Select id"}
```

Comment:
I think a placeholder of `None` will look better. Also, there's a warning in the console related to this select box:

![CleanShot 2022-12-01 at 21 13 44](https://user-images.githubusercontent.com/3920893/205150738-72e998be-7ac8-4422-bfb6-82d69ed0206f.png)

Reply:
Yes, it is related to inner `SelectField` implementation - I will take a look how to fix it